index.html

```diff
@@ -4,170 +4,223 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
-    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="./css/style.css">
     <title>Adriaville Contact</title>
 </head>
-
 <body>
-    <div class="contact">
-        <header>
-            <div class="logo-large">
-                <img src="images/logo green.svg">
-            </div>
-            <div class="header-center united-kingdom">
-                <img src="images/united-kingdom.png" srcset="img/united-kingdom@2x.png 2x, img/united-kingdom@3x.png 3x">
-            </div>
-            <div class="header-center">
-                <p class="eng">English</p>
-            </div>
-            <div class="header-center back">
-                <img src="images/back.png" srcset="img/back@2x.png 2x, img/back@3x.png 3x">
-            </div>
-            <div class="header-center btn-help">
-                <button class="btn-first">Help</button>
-            </div>
-            <div class="header-center btn-contact">
-                <button class="btn-second">Contact us</button>
+
+    <nav class="container-fluid py-4 mb-2">
+        <div class="row">
+
+            <div class="col-6">
+                <div class="mx-lg-5">
+                    <img src="images/logo green.svg">
+                </div>
             </div>
-            <div class="hamburger-large">
-                <img src="images/sidebar.svg">
+        
+            <div class="col-6">
+                <div class="d-none d-lg-flex justify-content-lg-around align-items-lg-center">
+                    <div class="d-none d-lg-block">
+                        <img src="images/united-kingdom.png" class="pr-2">
+                        <p class="eng m-0 d-lg-inline-flex">English</p>
+                        <img src="images/back.png" class="pl-2">
+                    </div>
+                    <div class="d-none d-lg-block">
+                        <button type="button" class="btn-first btn btn-primary">Help</button>
```

Comment:
Its very strange to switch from filled sandy color to outline blue. Dont change colors, just change the fill/no-fill variation.
Example:
non-hover state: green outline, white fill inside
hover state: green fill inside

etc.